gulpfile.babel.js

```diff
@@ -247,6 +249,19 @@ gulp.task('json', function() {
 });
 
 
+const accessAsync = promisify(fs.access);
+
+
+gulp.task('keycheck', () =>
+  accessAsync('./service-account-key.json')
```

Comment:
I think you need to return this promise if you want its rejection to fail the gulp task.

Reply:
The promise is returned; arrow functions without {} return their expression.

Reply:
Hmmm, I didn't notice it didn't have the braces because it was on a newline and indented as a block (e.g. 2 spaces instead of 4).

I don't think the intent is very clear here. I'd make it a block with an explicit return statement. What you save in conciseness you lose in clarify (IMO).

Reply:
Fixed w/ reindentation